docs/ja/syntax.md

```diff
@@ -0,0 +1,205 @@
+    <span v-show="validation.comment.maxLength">Your comment is too long.</span>
+  </div>
+  <input type="submit" value="send" v-if="valid">
+</form>
+```
+
+v2.0-alpha以降:
+```html
+<validator name="validation">
+  <form novalidate>
+    <input type="text" v-validate:comment="{ minlength: 16, maxlength: 128 }">
+    <div>
+      <span v-show="$validation.comment.minlength">Your comment is too short.</span>
+      <span v-show="$validation.comment.maxlength">Your comment is too long.</span>
+    </div>
+    <input type="submit" value="send" v-if="valid">
+  </form>
+</validator>
+```
+
+### キャメルケースプロパティ
+[Vue.js](https://jp.vuejs.org/guide/components.html#キャメルケース-対-ケバブケース) と同様に、ケバブケース( kebab-case : ハイフンで句切られた)を `v-validate` モデルに使用できます:
+
+```html
+<validator name="validation">
+  <form novalidate>
+    <input type="text" v-validate:user-name="{ minlength: 16 }">
+    <div>
+      <span v-if="$validation.userName.minlength">Your user name is too short.</span>
+    </div>
+  </form>
+</validator>
+```
+
+### 属性
+あなたはフィールド名を `field` パラメーター属性で指定することができます。 これはあなたがバリデーション可能なフォーム要素を動的に定義する際に便利です:
+
+> 注: `field` パラメーター属性を利用する場合は、フィールド部分の `v-validate` は任意です。
+
+```html
+<div id="app">
+  <validator name="validation">
+    <form novalidate>
+      <p class="validate-field" v-for="field in fields">
+      <label :for="field.id">{{field.label}}</label>
+      <input type="text" :id="field.id" :placeholder="field.placeholder" field="{{field.name}}" v-validate="field.validate">
+      </p>
+      <pre>{{ $validation | json }}</pre>
+    </form>
+  </validator>
+</div>
+```
+
+```javascript
+new Vue({
+  el: '#app',
+  data: {
+    fields: [{
+      id: 'username',
+      label: 'username',
+      name: 'username',
+      placeholder: 'input your username',
+      validate: { required: true, maxlength: 16 }
+    }, {
+      id: 'message',
+      label: 'message',
+      name: 'message',
+      placeholder: 'input your message',
+      validate: { required: true, minlength: 8 }
+    }]
+  }
+})
+```
+
+
+## リテラル
+
+### 配列
+下記の例では配列リテラルを使用しています:
+
+```html
+<validator name="validation">
+  <form novalidate>
+    Zip: <input type="text" v-validate:zip="['required']"><br />
+    <div>
+      <span v-if="$validation.zip.required">Zip code is required.</span>
+    </div>
+  </form>
+</validator>
+```
+
+`required` で追加のルールを指定する必要が無いので、この文法は望ましいです。
+
+
+### オブジェクト
+下記の例ではオブジェクトリテラルを使用しています:
+
+```html
+<validator name="validation">
+  <form novalidate>
+    ID: <input type="text" v-validate:id="{ required: true, minlength: 3, maxlength: 16 }"><br />
+    <div>
+      <span v-if="$validation.id.required">ID is required</span>
+      <span v-if="$validation.id.minlength">Your ID is too short.</span>
+      <span v-if="$validation.id.maxlength">Your ID is too long.</span>
+    </div>
+  </form>
+</validator>
+```
+
+オブジェクトリテラルはルールを与えることができます。上記で示すように、 `required` にはルールが必要なく、 **ダミールール**を代わりに指定することができます。
```

Comment:
At first, I couldn't understand the meaning of `dummy rule` in this part. But I could understand when I write a sample code like this `ID: <input type="text" v-validate:id="{ required: we_can_set_here_anything_because_it_is_dummy , minlength: 3, maxlength: 16 }">`.  It works anyway.